pkg/rtc/supervisor/participant_supervisor.go

```diff
@@ -154,6 +156,10 @@ func (p *ParticipantSupervisor) UpdateSubscription(trackID livekit.TrackID, isSu
 		},
 	)
 	p.lock.Unlock()
+
+	if isSubscribe {
+		prometheus.AddSubscribeAttempt(sourceTrack.Kind().String(), true)
```

Comment:
This is actually subscribe attempt. Not success yet.

Reply:
Moved this to into subscription_monitor to stay consistent with the others.